stream/stream_lavf.c

```diff
@@ -310,6 +310,10 @@ static int open_f(stream_t *stream)
         av_dict_set(&dict, "timeout", "0", 0);
     }
 
+    if (strncmp(filename, "srt", 3) == 0) {
+        stream->demuxer = "lavf";
```

Comment:
Are you sure this is required? I have mostly been poking with pure UDP and it moved nicely through probing to MPEG-TS for example.

I'd rather not add more kludges if it's possible :) .

Reply:
You are right, without the demuxer forced, MPV is still able to read my stream :)

If you want to merge something more clean I can close this MR and recreate a new branch with this modification.